chaosblade-exec-service/src/main/java/com/alibaba/chaosblade/exec/service/build/JavaHomeFlagBean.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 1999-2019 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.chaosblade.exec.service.build;
+
+import com.alibaba.chaosblade.exec.common.model.FlagSpec;
+
+/**
+ * @author Changjun Xiao
+ */
+public class JavaHomeFlagBean implements FlagSpec {
+
```

Comment:
what‘s the default value if user not set.

Reply:
The value is empty.  But the chaosblade will get the JAVA_HOME local variable or parse the command line to get the java path.